rudder-server-relay/debian/postinst

```diff
@@ -21,7 +21,7 @@ set -e
 case "$1" in
     configure)
 
-      SITES_TO_DISABLE="default default-ssl"
+      SITES_TO_DISABLE="default 000-default default-ssl"
```

Comment:
Ahhhh, I think what is happening here is that the *name* of the default site has been changed between Apache 2.2 and 2.4:

Apache 2.2:
sites-available/default.conf

Apache 2.4:
sites-available/000-default.conf

Therefore, one out of two of these disable commands will fail, because only one exists. However, that is fine, because the a2dissite command below is followed by ```|| true```